tests/benchdnn/doc/knob_cold_cache.md

```diff
@@ -14,25 +15,30 @@ end users typically have. For example, given an application's number of blocks
 and their sizes, most memory objects come from RAM and only some
 pieces roll over from the previous layer.
 
-In order to measure the library functionality closer to real-world
-applications, benchdnn has introduced a cold cache feature. In the core,
-each execution call in a performance measurement loop uses new memory objects
-for specific execution arguments, which is controlled by the user.
+To measure the library functionality closer to real-world applications, benchdnn
+has introduced a cold cache feature. In the core, each execution call in a
+performance measurement loop uses new memory objects for specific execution
+arguments, which is controlled by the user.
 
-To enable cold cache, users must specify `--cold-cache=MODE` and choose a
-`MODE` from one of three options: `wei`, `all`, or `custom` (lower case).
+## Usage
+```
+    --cold-cache=MODE[+EXTENSION]
+```
+
+`MODE` can be the one of these options: `none` (the default), `wei`, `all`, or
+`custom` (lower case).
 
 `wei` mode has benchdnn prepare a pile of weights tensors and, for each new
 run, use a new set until the stack is over. Then, it starts from the top of
-the pile again. It ensures that pile size is sufficient enough to evict
-previously used sets of weights from caches. This mode emulates a regular
-situation in topologies for primitives such as (de-)convolutions, inner
-product, and matmul. Usually, these primitives (and whole models) use a
-technique called "double buffering," where all activations transfer between
-two buffers: one for source and one for destination. It allows for an
-increased cache hit rate; only weights are taken from RAM. Note that if a
-primitive requests this mode but does not have a notion of weights, a warning
-is printed to stdout and cold cache is not enabled.
+the pile again. It ensures that pile size is sufficient to evict previously used
+sets of weights from caches. This mode emulates a regular situation in
+topologies for primitives such as (de-)convolutions, inner product, and matmul.
+Usually, these primitives (and whole models) use a technique called "double
+buffering," where all activations transfer between two buffers: one for source
+and one for destination. It allows for an increased cache hit rate; only weights
+are taken from RAM. Note that if a primitive requests this mode but does not
```

Comment:
```suggestion
It is common in modern frameworks that a primitive output activations reuse 
the same memory pages and cache lines as the input activations of the previously
run primitive. As a result, cache hit rate is often increased for activation 
accesses while weights are taken from RAM. Note that if a primitive requests 
this mode but does not
```

This is a consequence of memory allocation, not double-buffering.
FWKs usually do:
```
operation 1:
- allocate outputs
- run operation
- free inputs
operation 2:
- allocate outputs
- run operation
- free inputs
...
```
Depending on allocator implementation, the allocation of `operation 2 outputs` will use the same pages/cache lines as `operation 1 inputs`.